Skip_list/README.md

```diff
@@ -0,0 +1,112 @@
+# SKIP LIST
```

Comment:
I guess you have added Skip list README.md and skip_list.cpp by mistake as these files do not seem to be related to this PR. Please fix this.